### PR TITLE
Updated Extend section in Readme

### PR DESCRIPTION
**Pull request by Johnken-S**
Hi @jacalata 
Here goes the PR for Readme.  Also, When will be new release in 
https://www.npmjs.com/package/pseudoloc-js ?
Noticed that in Keywords : Pseudoloc' is wrongly spelt as 'psuedoloc'.

John

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -99,13 +99,15 @@ Default is `%`.
 #### Extend
 
 Extends the width of the string by the specified percentage. Useful if you will be localizing into languages such as German which can be 30% longer than English.
+We extend it by a set of predefined Asian characters.
```
Comment (MickMonaghanGW) [inline Readme.md]: They're not all Asian characters. How about:
`We extend the string by randomly selecting characters from a predefined array of non-ASCII characters`

File: Readme.md [hunk 2]
```diff
@@ -99,13 +99,15 @@ Default is `%`.
...
 
 Default is `0`.
 
     pseudoloc.option.extend = 0.3; //30%
     pseudoloc.str('A test string with a %token%.')
-    // [!!Ȃ ťēšť ŝťŕĩŉğ ŵĩťħ â %token%.        !!]
+    // [!!Ȃ ťēšť ŝťŕĩŉğ ŵĩťħ â %token%. öఛฒそ!!]
 
+New update: Extends the width of the string by the specified percentage will be filled with Asian characters including emoji.
```
Comment (MickMonaghanGW) [inline Readme.md]: The emoji character is not necessarily part of the extended string - they're randomly chosen from the array declared in the `pad` function.

**Review by jacalata — CHANGES_REQUESTED** [review]
I think Mick's suggestions are good, can you include them? I'll update the npm package once this readme is in.

**Comment by Johnken-S** [comment]
@jacalata  I have now updated based on @MickMonaghan  suggesstions.

**Comment by MickMonaghanGW** [comment]
Do we also need to replace instances of `pseudoloc` with `pseudoloc-js`? - such as in the `require` statement etc.?

**Review by jacalata — APPROVED** [review]

**Comment by jacalata** [comment]
I don't think there's any updates required for that, the package name is pseudoloc-js but the filename that require looks for is still just pseudoloc.  But I haven't run that example code recently - was it not working somwhere?

**Comment by MickMonaghanGW** [comment]
Ah, ok. Not an issue then. Thanks.

**Comment by Johnken-S** [comment]
Hi @jacalata : 
Can you please update the npm pseudoloc-js package with latest code changes?
Thank you!

**Comment by jacalata** [comment]
done, sorry for the delay! Also fixed a test that had been broken by the double-encoding padded chars (because javascript is so lame at string length)

**Comment by Johnken-S** [comment]
Thank you, @jacalata .

**Pull request merged**